openapi/main.yaml

```diff
@@ -0,0 +1,704 @@
+openapi: 3.0.3
+info:
+  title: Wallet Backend API
+  version: 1.0.0
+  description: |
+    The Wallet Backend has a set of common services exposed via an API that wallet applications built on top of the Stellar network can use. 
+
+    This is an open-source project that is built on top of the Stellar network, and it's code can be found in [stellar/wallet-backend](https://github.com/stellar/wallet-backend)
+  termsOfService: https://stellar.org/terms-of-service
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: ''
+tags:
+  - name: Account Registration
+    x-displayName: Account Registration
+    description: Clients of the Wallet Backed API can register and de-register stellar accounts whose payments they want to track
```

Comment:
So to confirm, registration is only relevant to the data returned from `/payments`, right? These accounts are irrelevant to the `tx` and `tss` endpoints?

Reply:
It actually also applies to the create-fee-bump-tx endpoint - ie: only transactions with stellar accounts that are registered with the wallet backend as the source can call the create-fee-bump-tx endpoint. We may need to revisit that logic though